test/bigtest/network/boot.js

```diff
@@ -0,0 +1,8 @@
+import startMirage from './start';
```

Comment:
@K-Felk could we do something similar to:

https://github.com/folio-org/ui-users/blob/master/test/bigtest/network/boot.js

here?

That way the `start.js` file could be also removed from this PR.

Reply:
Works fine, done, and removing start.js